### PR TITLE
User interaction with progress package kevva/brightness#1

### PR DESCRIPTION
**Pull request by ewnd9**
for kevva/brightness#1

### CODE REVIEW

**Comment by kevva** [comment]
Thanks, will review this this afternoon.

**Comment by ewnd9** [comment]
I've extracted control functionality into separate [progress-control](https://github.com/ewnd9/progress-control) package in order to contribute to gillstrom/vol/issues/2

File: cli.js [hunk 1]
```diff
@@ -13,13 +14,47 @@ var cli = meow({
...
 
-			console.log(data);
+			// check permissions
+			brightness.set(value, function (err) {
+				if (err) {
+					console.error(err.message);
+					process.exit(1);
+				}
+
+				var updateBar = function(value) {
```
Comment (kevva) [inline cli.js]: `function updateBar(val) {`

File: cli.js [hunk 2]
```diff
@@ -13,13 +14,47 @@ var cli = meow({
...
+						}
+					});
+
+					bar.update(value, {
+						value: value
+					});
+				};
+
+				var bar = progressControl('Use up/down arrows [:bar] :value ', { total: 10 }, {
+				  'up': function() {
```
Comment (kevva) [inline cli.js]: `up: function () {`

File: cli.js [hunk 3]
```diff
@@ -13,13 +14,47 @@ var cli = meow({
...
+						value = Math.min(Math.round((value + 0.1) * 10) / 10, 1);
+						updateBar(value);
+				  },
+				  'down': function() {
```
Comment (kevva) [inline cli.js]: `down: function () {`

**Comment by kevva** [comment]
Thanks for this :). Great work.

**Comment by gillstrom** [comment]
:+1: :+1: :+1:

**Pull request closed without merging**